website/public/errors/index.ejs.html

```diff
@@ -47,12 +47,24 @@
         text-decoration: none;
       }
 
-      .unicode-bom:before {
-        content: "BOM";
-        border: 1px solid red;
-        background-color: #884444;
-        border-radius: 5px;
-        padding: 1px 3px;
+      @media (prefers-color-scheme: dark) {
+        .unicode-bom:before {
+            content: "BOM";
+            border: 1px solid red;
+            background-color: #884444;
+            border-radius: 5px;
+            padding: 1px 3px;
+          }
+      }
+
+      @media (prefers-color-scheme: light) {
+        .unicode-bom:before {
+          content: "BOM";
+          border: 1px solid red;
+          background-color: #edbf66;
+          border-radius: 5px;
+          padding: 1px 3px;
+        }
       }
```

Comment:
Assume that light mode is the default, in case `prefers-color-scheme` is not implemented:
```suggestion
        .unicode-bom:before {
          content: "BOM";
          border: 1px solid red;
          background-color: #edbf66;
          border-radius: 5px;
          padding: 1px 3px;
        }
```